### PR TITLE
feat: add new translations for Vietnamese, Tagalog, and Chinese

### PR DESCRIPTION
**Pull request by jaredcwhite**
## Issue Overview

This PR addresses #494 

- [ ] This change addresses the issue in full
- [x] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

This updates the locale JSON files, but does not address the backend emails. I need to confer with the rest of the team what's the best way to work on that in the new Prisma context, and either update this PR or file something separately.

## How Can This Be Tested/Reviewed?

Look at various pages in the Public app and see if new translations show up, particularly when browsing for listings.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: shared-helpers/src/locales/tl.json [hunk 1]
```diff
@@ -340,10 +340,10 @@
   "authentication.signIn.changeYourPassword": "Maaari mong palitan ang iyong password",
   "authentication.signIn.enterLoginEmail": "Pakilagay ang iyong email sa pag-login",
   "authentication.signIn.enterLoginPassword": "Pakilagay ang iyong password sa pag-log in",
-  "authentication.signIn.enterValidEmailAndPasswordAndMFA": "Pakilagay ang tamang code.",
   "authentication.signIn.enterValidEmailAndPassword": "Pakilagay ang tamang email at password.",
-  "authentication.signIn.errorGenericMessage": "Pakisubukang muli_ o makipag-ugnayan sa suporta para humingi ng tulong.",
+  "authentication.signIn.enterValidEmailAndPasswordAndMFA": "Pakilagay ang tamang code.",
   "authentication.signIn.error": "Nagkaproblema sa iyong pag-sign in",
+  "authentication.signIn.errorGenericMessage": "Pakisubukang muli_ o makipag-ugnayan sa suporta para humingi ng tulong.",
```
Comment (emilyjablonski) [inline shared-helpers/src/locales/tl.json:346]: This was fixed recently in core, but somehow we've had a lot of underscores get in where commas are supposed to be, it was easy to find and place them with commas, there were only a handful of cases in tags where the underscore was needed.

Comment (jaredcwhite) [reply shared-helpers/src/locales/tl.json:346]: Ah yeah, I saw the convo about that and forgot to double-check here. Will take a look.

File: shared-helpers/src/scripts/import-new-translations.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/no-var-requires, import/no-unresolved */
+
+// Usage:
+// ts-node --skip-project src/scripts/import-new-translations.ts tl Tagalog.csv
+//
+// Requires an input CSV with three columns: English, Translation, and (optionally) Updated Translation
```
Comment (emilyjablonski) [inline shared-helpers/src/scripts/import-new-translations.ts:6]: What's an example here?
"I'm a string" "I'm a string in a foreign language" "I'm a string in a foreign language updated"?
I'm not clear what the updated translation would be compared to the translation itself

Comment (jaredcwhite) [reply shared-helpers/src/scripts/import-new-translations.ts:6]: @emily Yeah, I think it's a bit odd too, but that's the format of the spreadsheets/CSVs we got so I had to build that into the script.

**Review by emilyjablonski — APPROVED** [review]

**Pull request merged**